doc/nrf/known_issues.rst

```diff
@@ -2000,7 +2001,7 @@ Secure Partition Manager (SPM)
 .. note::
     The Secure Partition Manager (SPM) is deprecated as of |NCS| v2.1.0 and removed after |NCS| v2.2.0. It is replaced by :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`.
 
-.. rst-class:: v2-2-0
+.. rst-class:: v2-3-0 v2-2-0
 
 NCSDK-19156: Building SPM for other boards than ``nrf5340dk_nrf5340_cpuapp`` and ``nrf9160dk_nrf9160`` fails with compilation error in cortex_m_systick.c
```

Comment:
SPM was removed in v2.2.0 so this is not an issue in v2.3

Reply:
Removed tag for NCSDK-19156 and NCSIDB-114, @osaether .